music/src/core/sequences.ts

```diff
@@ -0,0 +1,290 @@
+/**
```

Comment:
Is there a way to force this to be seen as a branch of the old sequences.ts?

Reply:
I don't think there's an easy way. Git is supposed to detect these things automatically. I could try going back in the commit history and doing an explicit move as its own commit and then modify in other commits, but that's going to be a pain at this point.